report_artifact.go

```diff
@@ -172,38 +178,44 @@ func (a *TestArtifact) bsonToFTDC(ctx context.Context, path string) (string, err
 func (a *TestArtifact) csvToFTDC(ctx context.Context, path string) (string, error) {
 	srcFile, err := os.Open(path)
 	if err != nil {
-		return path, errors.Wrapf(err, "problem opening csv input file '%s'", path)
+		return path, errors.Wrapf(err, "opening CSV input file '%s'", path)
 	}
-	defer srcFile.Close()
+	defer func() {
+		_ = srcFile.Close()
+	}()
 
 	path = strings.TrimSuffix(path, ".csv") + ".ftdc"
 	catcher := grip.NewCatcher()
 	ftdcFile, err := os.Create(path)
 	if err != nil {
-		return path, errors.Wrapf(err, "problem opening ftdc output file '%s'", path)
+		return path, errors.Wrapf(err, "opening ftdc output file '%s'", path)
 	}
-	defer func() { catcher.Add(ftdcFile.Close()) }()
+	defer func() {
+		_ = ftdcFile.Close()
+	}()
 
-	catcher.Add(errors.Wrap(ftdc.ConvertFromCSV(ctx, defaultChunkSize, srcFile, ftdcFile),
-		"problem converting csv to ftdc file"))
+	catcher.Wrap(ftdc.ConvertFromCSV(ctx, defaultChunkSize, srcFile, ftdcFile), "converting CSV to FTDC file")
```

Comment:
I don't actually think we need to use a catcher here, we can just return this error.